public/index.html

```diff
@@ -27,7 +27,9 @@
     type="image/x-icon" />
   <link rel="icon" href="https://cdn.coronasafe.network/care-manifest/images/icons/favicon.ico" type="image/x-icon" />
   <link rel="manifest" href="/manifest.webmanifest" crossorigin="use-credentials" />
-  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
```suggestion
```
Why do we need this 🤔

Reply:
@bodhish isn't it a standard practice to preconnect to google fonts for faster loads?

https://www.cdnplanet.com/blog/faster-google-webfonts-preconnect/

Should I still remove it?

Reply:
![image](https://user-images.githubusercontent.com/25143503/194465587-2311d1e0-86af-440b-9b0c-42195a590f85.png)